notebooks/image_models/solutions/serving.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<p>Maybe double check the plurals and capitalization: e.g.,</p><p> </p><ul><li>Serving model -&gt; Serving Models</li><li>Learn how to export TensorFlow model -&gt; Learn how to export TensorFlow models</li><li>Learn how to customize signature -&gt; Learn how to customize signatures</li><li>Learn how to use deployed model in online and batch prediction -&gt; Learn how to use a deployed model in online and batch prediction </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thank you for the correction. Modified!</p>